src/sentry/deletions/defaults/group.py

```diff
@@ -132,48 +100,8 @@ def tenant_ids(self) -> Mapping[str, Any]:
     def chunk(self) -> bool:
         """This method is called to delete chunks of data. It returns a boolean to say
         if the deletion has completed and if it needs to be called again."""
-        if not options.get("deletions.nodestore.parallelization-task-enabled"):
-            events = self.get_unfetched_events()
-            if events:
-                # Adding this variable to see the values in stack traces
-                last_event = events[-1]
-                self.delete_events_from_nodestore_sequential(events)
-                # This value will be used in the next call to chunk
-                self.last_event = last_event
-                # As long as it returns True the task will keep iterating
-                return True
-            else:
-                # Now that all events have been deleted from the eventstore, we can delete the events from snuba
-                self.delete_events_from_snuba()
-                return False
-        else:
-            self.delete_events_from_nodestore_and_eventstore()
-            return False
-
-    # XXX: To be removed
-    def delete_events_from_nodestore_sequential(self, events: Sequence[Event]) -> None:
-        # We delete by the occurrence_id instead of the event_id
-        node_ids = [
-            Event.generate_node_id(
-                event.project_id,
-                (
-                    event._snuba_data["occurrence_id"]
-                    if self.dataset == Dataset.IssuePlatform
-                    else event.event_id
-                ),
-            )
-            for event in events
-        ]
-        nodestore.backend.delete_multi(node_ids)
-        self.post_delete_events_from_nodestore(events)
-
-    # XXX: To be removed
-    def post_delete_events_from_nodestore(self, events: Sequence[Event]) -> None:
-        pass
-
-    # XXX: To be removed
-    def delete_events_from_snuba(self) -> None:
-        raise NotImplementedError
+        self.delete_events_from_nodestore_and_eventstore()
```

Comment:
This function call is what handles the new task scheduling and deprecates all the other code.